### PR TITLE
Remove outdated and not maintained examples

### LINKED ISSUE #3450: refactor commit destroyed image-handling example
Commit [9cfcc80f4af6d9e95684574d2730cd34522fdd36](https://github.com/sequelize/sequelize/commit/9cfcc80f4af6d9e95684574d2730cd34522fdd36#diff-be5a43cf3e463220455877c3ad75e126) replaced examples/image-handling/app.js with examples/default-values/app.js 

```diff
diff --git a/examples/image-handling/app.js b/examples/image-handling/app.js
index b7c0ab9..4b774b7 100644
--- a/examples/image-handling/app.js
+++ b/examples/image-handling/app.js
@@ -1,21 +1,28 @@
-var fs        = require("fs")
-  , Sequelize = require("sequelize")
-  , sequelize = new Sequelize('sequelize_test', 'root', null, {logging: false})
-  , Image     = sequelize.define('Image', { data: Sequelize.TEXT })
+/*
+  Title: Default values

-Image.sync({force: true}).on('success', function() {
-  console.log("reading image")
-  var image = fs.readFileSync(__dirname + '/source.png').toString("base64")
-  console.log("done\n")
-
-  console.log("creating database entry")
-  Image.create({data: image}).on('success', function(img) {
-    console.log("done\n")
-
-    console.log("writing file")
-    fs.writeFileSync(__dirname + '/target.png', img.data, "base64")
-    console.log("done\n")
-
-    console.log("you might open the file ./target.png")
+  This example demonstrates the use of default values for defined model fields. Instead of just specifying the datatype,
+  you have to pass a hash with a type and a default. You also might want to specify either an attribute can be null or not!
+*/
+
+var Sequelize = require(__dirname + "/../../index")
+  , config    = require(__dirname + "/../../spec/config/config")
+  , sequelize = new Sequelize(config.database, config.username, config.password, {logging: false})
+
+var User = sequelize.define('User', {
+      name: { type: Sequelize.STRING, allowNull: false},
+      isAdmin: { type: Sequelize.BOOLEAN, allowNull: false, defaultValue: false }
+    })
+  , user = User.build({ name: 'Someone' })
+
+sequelize.sync({force: true}).on('success', function() {
+  user.save().on('success', function(user) {
+    console.log("user.isAdmin should be the default value (false): ", user.isAdmin)
+
+    user.updateAttributes({ isAdmin: true }).on('success', function(user) {
+      console.log("user.isAdmin was overwritten to true: " + user.isAdmin)
+    })
   })
+}).on('failure', function(err) {
+  console.log(err)
 })
```

### PR DESCRIPTION
**Pull request by BridgeAR**
These confuse people and some think they can rely on them. 

This closes #3450

### CODE REVIEW

**Comment by janmeier** [comment]
Hmm, I think I agree that this should be removed. @mickhansen ?

**Comment by mickhansen** [comment]
Ideally we should provide better ones, but yeah ;p

**Comment by BridgeAR** [comment]
Seems like it's ready to merge? :)

File: examples/postgres-json-datatype/app.js [hunk 1]
```diff
@@ -1,84 +0,0 @@
-/*
-  Title: PostgreSQL JSON Data-Type
-
-  An example of using PostgreSQL's JSON data-type.
-  In this example we create a single table that can handle multiple types of different media and it's metadata.
-
-  This example uses the promise API preferred in 2.0 and above.
-*/
-
-var Sequelize = require(__dirname + "/../../index")
```
Comment (mickhansen) [inline examples/postgres-json-datatype/app.js]: this example actually isnt that outdated.

**Comment by mickhansen** [comment]
The JSON example wasnt outdated, but doesn't promote the best syntax.

**Comment by BridgeAR** [comment]
If sequelize wants to have examples it's probably also better not to only have a single up to date one ;) In my opinion the docs are getting better and better every day and people should look at them in the beginning. If they find they examples they might be tempted not to.

**Comment by mickhansen** [comment]
Yeah, might be better to have examples on http://runnable.com/ too.

**Pull request merged**